udata_recommendations/tasks.py

```diff
@@ -106,11 +108,23 @@ def process_dataset(source, dataset):
             except (Reuse.DoesNotExist, mongoengine.errors.ValidationError):
                 error(f"Recommended reuse {reco['id']} not found")
                 continue
+        elif reco_type == 'external':
+            try:
+                external = urlsplit(reco['id']).geturl()
```

Comment:
Do you use it to validate the url? You could use the helper `from udata.uris import validate` if we want to be consistent on our URL validation?

Reply:
Thx !